labrad/types/__init__.py

```diff
@@ -0,0 +1 @@
+from labrad.types.types import *
```

Comment:
Let's see how this looks -- the formatting of the imports is a bit redundant, but I did this to group things logically together.